web3/_utils/method_formatters.py

```diff
@@ -486,6 +487,60 @@ def apply_list_to_array_formatter(formatter: Any) -> Callable[..., Any]:
 ABI_REQUEST_FORMATTERS = abi_request_formatters(STANDARD_NORMALIZERS, RPC_ABIS)
 
 
+def get_revert_reason(response: RPCResponse) -> str:
```

Comment:
It feels like this `method_formatters` file is becoming a catch-all, so may be worth thinking about how we can split this out. But not for this PR :)